pandas/tests/test_window.py

```diff
@@ -520,6 +520,19 @@ def test_closed(self):
         with pytest.raises(ValueError):
             df.rolling(window=3, closed='neither')
 
+    @pytest.mark.parametrize("closed", ["neither", "left"])
+    @pytest.mark.parametrize(
+        "func", ["std", "mean", "median", "sum", "max", "min", "var"])
+    def test_closed_empty(self, closed, func):
```

Comment:
we have a fixture for these - look in pandas/conftest

Reply:
I guess it is `all_numeric_reductions` but `prod` function fails since it is not implemented for `Rolling`. I will add `kurt` and `skew` to the list

Reply:
you may need to define a separate fixture to avoid some issues

Reply:
Done